### PR TITLE
enable deamonset extensions in e2e test clusters

### PR DESCRIPTION
**Pull request by mikedanese**
Otherwise not all e2e tests will work

### CODE REVIEW

**Comment by brendandburns** [comment]
LGTM.  Will merge on green for build fixes.

File: cluster/gce/config-default.sh [hunk 1]
```diff
@@ -106,7 +106,12 @@ fi
 # Optional: Enable deployment experimental feature, not ready for production use.
 ENABLE_DEPLOYMENTS="${KUBE_ENABLE_DEPLOYMENTS:-false}"
 if [[ "${ENABLE_DEPLOYMENTS}" == "true" ]]; then
-  ENABLE_EXPERIMENTAL_API=true
+    ENABLE_EXPERIMENTAL_API=true
```
Comment (ixdy) [inline cluster/gce/config-default.sh]: nit: this file seems to use 2-space indent, not 4

File: cluster/gce/config-test.sh [hunk 2]
```diff
@@ -119,6 +119,12 @@ if [[ "${ENABLE_DEPLOYMENTS}" == "true" ]]; then
   ENABLE_EXPERIMENTAL_API=true
 fi
 
+# Optional: Enable daemonset experimental feature, not ready for production use.
+ENABLE_DAEMONSETS="${KUBE_ENABLE_DAEMONSETS:-true}"
+if [[ "${ENABLE_DAEMONSETS}" == "true" ]]; then
+    ENABLE_EXPERIMENTAL_API=true
```
Comment (ixdy) [inline cluster/gce/config-test.sh]: same

**Comment by ixdy** [comment]
LG other than nits

**Comment by mikedanese** [comment]
@ixdy fixed

File: cluster/gce/config-default.sh [hunk 3]
```diff
@@ -106,6 +106,11 @@ fi
 # Optional: Enable deployment experimental feature, not ready for production use.
 ENABLE_DEPLOYMENTS="${KUBE_ENABLE_DEPLOYMENTS:-false}"
 if [[ "${ENABLE_DEPLOYMENTS}" == "true" ]]; then
+    ENABLE_EXPERIMENTAL_API=true
```
Comment (ixdy) [inline cluster/gce/config-default.sh]: this one is still 4 spaces? (nittiest nit)

**Comment by mikedanese** [comment]
@ixdy fixed. stupid emacs formatting.

**Comment by ixdy** [comment]
LGTM, thanks

**Comment by ixdy** [comment]
sorry, failure caused by restarting Jenkins.
@k8s-bot test this please

**Comment by mikedanese** [comment]
@k8s-bot ok to test

**Pull request merged**